src/components/Zoom/utils/useActiveShareChange.ts

```diff
@@ -0,0 +1,28 @@
+import { RefObject } from 'react';
+
+const useActiveShareChange = (
+  client: any,
```

Comment:
what is it?

Reply:
This is Zoom SDK property. I didnt find the way how apply type for it.